_summaries/image-tiff.md

```diff
@@ -0,0 +1,5 @@
+---
+name: "tiff"
+---
+
+Decoder for the `tiff` file format.
```

Comment:
Remember that this now is both an encoder and decoder.

Reply:
Right, I'll amend it.